lighthouse-core/formatters/table.js

```diff
@@ -78,13 +78,18 @@ class Table extends Formatter {
 
     const rows = results.map(result => {
       const cols = headingKeys.map(key => {
+        let value = result[key];
+        if (typeof value === 'undefined') {
```

Comment:
Oh the fix for that is most likely the `~` added to the report HTML, that was some more aggressive edge case hunting, but yes we totally should. I'm in favor of getting this consistency PR through between the byte efficiency and then doing a bit of refactor cleanup of the stuff that's starting to be repeated everywhere, if that makes sense to you too?